todo/src/Components/TodoClicks.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Checkbox, IconButton, ListItem, Typography } from "@material-ui/core";
+import CloseIcon from "@material-ui/core";
+
+function Todo({ todo, toggleComplete, removeTodo }) {
```

Comment:
I like this implementation.  It feels funny some how, but clear, and highly cohesive!